doc/process/joining_the_build_wg.md

```diff
@@ -24,3 +24,4 @@ request.
 [IRC]: /README.md#nodejs-build-working-group
 [the Readme]: /README.md
 [the onboarding doc]: /ONBOARDING.md
+[access.md]: ./doc/access.md
```

Comment:
Should be `/doc/access.md`, otherwise it tries to find [/doc/process/doc/access.md](https://github.com/nodejs/build/blob/master/doc/process/doc/access.md), which doesn't work.

Reply:
Good point. I just copied it from the root README. I was going to change that one to absolute, but I figured I'd minimize the diff